content/pieces/logging-off/_index.md

```diff
@@ -1,2 +1,4 @@
 ---
+prev_piece: ../../uncivilizing-digital-territories
```

Comment:
this is really not an ideal fix but it works 😕

Reply:
Great! This is how we know the product is about ready to ship!

Reply:
@benhylau as I was writing out a response about why I had to do this janky workaround, I figured out a better way to do it, so thanks for being my rubber duck!